client/state/comments/actions.js

```diff
@@ -221,6 +222,13 @@ export function writeComment( commentText, siteId, postId, parentCommentId ) {
 				skipSort
 			} );
 
+			dispatch( {
+				type: COMMENTS_COUNT_INCREMENT,
+				siteId,
+				postId,
+				incrementBy: 1
```

Comment:
If someone else added a comment in the meantime, that comment would not be displayed in the comment list and it would not count towards the comment total. If the page is refreshed, then our components would get and display that information.

This is a limitation of `PostCommentList` since it only fetches the full list of comments when loading for the first time, and does not fetch a new list when a new comment is added. 

The ultimate source of truth for the comment count is `state.comments.totalCommentsCount`. The comment list and comment total are not currently being kept in sync with the database after a new comment is made, but the comment count matches with the actual comments shown on the page.

In the future we could keep the state in sync with the database by either (1) pulling new data on every comment submission or (2) polling it for data periodically.